src/pkg/utils/helpers/url_test.go

```diff
@@ -90,6 +90,29 @@ func (suite *TestURLSuite) Test_2_DoHostnamesMatch() {
 	suite.False(b)
 }
 
+func (suite *TestURLSuite) Test_3_ExtractFilenameFromURL() {
+	urls := []string{
```

Comment:
We should have some things that will generate errors as well to ensure those return properly